package.json

```diff
@@ -34,6 +34,7 @@
     "@aws-sdk/client-lambda": "3.540.0",
     "@aws-sdk/client-sqs": "3.540.0",
     "@hathor/wallet-lib": "0.39.0",
+    "@wallet-service/common": "1.5.0",
```

Comment:
question(non-blocking): Will this require us to publish the wallet service common on `npm`? Or will `yarn` be able to identify that this package is not to be taken from an external source?

This is not clear to me because this is the root `package.json`, not a sub module.

Reply:
It won't, yarn will recognize it is a internal package and symlink it from the package